### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by eggplantzzz**
Update the docs to use the newer org-scoped drizzle packages instead of the old ones.

### CODE REVIEW

File: packages/react-plugin/README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ If you are using React v16.3 and up, you have access to the new [React Context](
 1. Setup Drizzle and then pass the `drizzle` instance into the context provider:
 
     ```js
-    // 1. Import drizzle, drizzle-react, and your contract artifacts.
+    // 1. Import drizzle, @drizzle/react-plugin, and your contract artifacts.
     import { Drizzle, generateStore } from "drizzle";
```
Comment (krzysu) [inline packages/react-plugin/README.md]: you could update also this line to 
```
import { Drizzle, generateStore } from "@drizzle/store";
```

**Comment by krzysu** [comment]
@adrianmcli is there any official way to contribute to your documentation as current state is not usable at all. Can we merge at least this PR soon or extend it to update other readmes in the project?

File: packages/react-plugin/README.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ Just like with the other approaches, you'll need to wrap your app in a context p
 
 ```js
 import React from 'react'
-import { drizzleReactHooks } from 'drizzle-react'
+import { drizzleReactHooks } from '@drizzle/react-plugin'
 import setUpDrizzle from './set-up-drizzle'
 import App from './app'
```
Comment (krzysu) [inline packages/react-plugin/README.md:264]: This example could be improved to:
```
import React from "react";
import ReactDOM from "react-dom";
import {Drizzle} from "@drizzle/store";
import {drizzleReactHooks} from "@drizzle/react-plugin";
import SimpleStorage from "./contracts/SimpleStorage.json";
import App from "./App";

const options = {contracts: [SimpleStorage]};
const drizzle = new Drizzle(options);
const {DrizzleProvider} = drizzleReactHooks;

ReactDOM.render(
  <DrizzleProvider drizzle={drizzle}>
    <App />
  </DrizzleProvider>,
  document.getElementById("root")
);
```

File: packages/react-plugin/README.md (hunk 1 above)
Comment (eggplantzzz) [reply packages/react-plugin/README.md]: Thanks @krzysu!

**Comment by adrianmcli** [comment]
lgtm, I missed this previously. Thanks!

**Pull request merged**